WcaOnRails/app/views/layouts/application.html.erb

```diff
@@ -87,5 +87,6 @@
   <% if !hide_wca_navbars %>
     <%= render 'layouts/footer' %>
   <% end %>
+  <%= render 'cookies_eu/consent_banner' %>
```

Comment:
```suggestion
  <%= render 'cookies_eu/consent_banner' if Rails.env.production? %>
```
This way it would show up on prod and staging, but would not mess up the tests (for which we honestly don't mind not having that banner).

Reply:
Thank you Philippe, that is a very smart and concise solution! Saved me a ton of trouble editing the tests.